doc/orchestrator.rst

```diff
@@ -165,3 +165,399 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+
+When you execute the program the following output will appear.
+
+.. code-block:: bash
+
+  21:01:54 osprey.us.cray.com SmartSim[105901] INFO The colo array key exists in the incorrect database: False
+  21:01:54 osprey.us.cray.com SmartSim[105901] INFO The clus array key exists in the incorrect database: False
+
+In the next stage of the workflow, we are going to write an application
+that utilizes the previously launched databases. In SmartSim terms, the application
+is called a ``Model``.
+
+To run a software program, you must specify the run settings: run command and
+source code file. In this example, we invoke the Python interpreter to run a
+python script. Create the run settings using the ``create_run_settings()`` function
+provided by the ``exp`` (``Experiment``) object. Specify the path to the source code file (script) to the argument
+``exe_args`` and the run command to the ``exe`` argument.
+
+.. code-block:: python
+
+  model_settings = exp.create_run_settings(exe="python", exe_args="model_multidb_example.py")
+
+Next, set the number of nodes and tasks per node for the ``Model``. For the application (``Model``),
+the number of nodes determines the distribution of computational tasks while tasks per node
+specifies how many individual instances of the ``Model`` (tasks) can run on each node. In this
+example, we specify to SmartSim that we intend to execute the ``Model`` once on a single node.
+
+.. code-block:: python
+
+  model_settings.set_nodes(1)
+  model_settings.set_tasks_per_node(1)
+
+Next, create an instance of the ``Model`` object using the ``create_model()`` function of the
+``exp`` object. Pass in the ``model_settings`` object, which was configured earlier, as an argument
+to the ``create_model()`` function and assign to the variable ``model``.
+
+.. code-block:: python
+
+  model = exp.create_model("tutorial-model", model_settings)
+  exp.generate(model_example, overwrite=True)
+
+Next, launch the model instance using the exp.start() function.
+
+.. code-block:: python
+
+  exp.start(model, block=True, summary=True)
+  exp.generate(model, overwrite=True)
+
+
+Finally, use the ``Experiment.stop()`` function to stop the database instances and print the
+workflow summary with ``exp.summary()``.
+
+.. code-block:: python
+
+  exp.stop(colocated_orch, clustered_orch)
+  logger.info(exp.summary())
+
+We have finished writing the experiment workflow. Continue reading to write the application workflow.
+
+Integrating SmartRedis Multi-Database Functionality into the Model
+------------------------------------------------------------------
+
+Next in the example, add SmartRedis multi-database functionality to the ``Model``.
+Navigate to the file specified in the ``exe_args`` argument of the ``create_run_settings()``
+function, ``exe_args="/path/to/model_multidb_example.py"``.
+
+To begin, learn to retrieve data from the launched databases by using a SmartRedis Client
```

Comment:
I think the goal in this step is not to retrieve data, but to learn how to connect a client to the database